### PR TITLE
Add French version of run scripts tutorial

### PR DESCRIPTION
**Pull request by jadechoghari**
## What does this PR do?

This pull request adds a French translation of the run_scripts from English to French.

## Changes made

- Translated `run_scripts.md` from English to French.
- Updated the `_toctree.yml` to include the French version of the run_scripts tutorial.

## Why is this change important?

Providing documentation in multiple languages helps make the information accessible to a broader audience and supports users who are more comfortable reading in French. 🇫🇷
More docs to come. 🤗

## Review

Like last time I will tag:
- @amyeroberts (for the merge ✅)
- @younesbelkada (for the review 🇫🇷🎊)

### CODE REVIEW

**Comment by jadechoghari** [comment]
seems younes is off, will cc @ArthurZucker 🇫🇷 for this PR.

**Review by ebrisk-bit — APPROVED** [review]
Semble bien 🇫🇷🎊

**Comment by jadechoghari** [comment]
@NielsRogge, could you kindly confirm the French part? 
We just need one review from a maintainer with write access to close this small PR, been 5 days. 
🤗 Thank you! 🇫🇷

File: docs/source/fr/run_scripts_fr.md [hunk 1]
```diff
@@ -0,0 +1,355 @@
...
+rendered properly in your Markdown viewer.
+
+-->
+
+# Entraîner avec un script
+
+En plus des [notebooks](./notebooks) de 🤗 Transformers, il existe également des exemples de scripts démontrant comment entraîner un modèle pour une tâche avec [PyTorch](https://github.com/huggingface/transformers/tree/main/examples/pytorch), [TensorFlow](https://github.com/huggingface/transformers/tree/main/examples/tensorflow) ou [JAX/Flax](https://github.com/huggingface/transformers/tree/main/examples/flax).
+
+
+Vous trouverez également des scripts que nous avons utilisés dans nos [projets de recherche](https://github.com/huggingface/transformers/tree/main/examples/research_projects) et des [exemples "legacy"](https://github.com/huggingface/transformers/tree/main/examples/legacy) qui sont des contributions de la communauté. Ces scripts ne sont pas activement maintenus et nécessitent une version spécifique de 🤗 Transformers qui sera probablement incompatible avec la dernière version de la bibliothèque.
```
Comment (ArthurZucker) [inline docs/source/fr/run_scripts_fr.md:22]:
```suggestion
Vous trouverez également des scripts que nous avons utilisé dans nos [projets de recherche](https://github.com/huggingface/transformers/tree/main/examples/research_projects) et des [exemples "legacy"](https://github.com/huggingface/transformers/tree/main/examples/legacy) qui sont des contributions de la communauté. Ces scripts ne sont pas activement maintenus et nécessitent une version spécifique de 🤗 Transformers qui sera probablement incompatible avec la dernière version de la librairie.
```

File: docs/source/fr/run_scripts_fr.md [hunk 2]
```diff
@@ -0,0 +1,355 @@
...
+
+Pour toute fonctionnalité que vous souhaitez implémenter dans un script d'exemple, veuillez en discuter sur le [forum](https://discuss.huggingface.co/) ou dans une [issue](https://github.com/huggingface/transformers/issues) avant de soumettre une Pull Request. Bien que nous acceptions les corrections de bugs, il est peu probable que nous fusionnions une Pull Request (opération "merge" dans Git) ajoutant plus de fonctionnalités au détriment de la lisibilité.
+
+Ce guide vous montrera comment exécuter un script d'entraînement de résumé en exemple avec [PyTorch](https://github.com/huggingface/transformers/tree/main/examples/pytorch/summarization) et [TensorFlow](https://github.com/huggingface/transformers/tree/main/examples/tensorflow/summarization). Tous les exemples sont censés fonctionner avec les deux frameworks, sauf indication contraire.
+
+## Configuration
+
+Pour exécuter avec succès la dernière version des scripts d'exemple, vous devez **installer 🤗 Transformers à partir de la source** dans un nouvel environnement virtuel :
```
Comment (ArthurZucker) [inline docs/source/fr/run_scripts_fr.md:32]:
```suggestion
Pour exécuter avec succès la dernière version des scripts d'exemple, vous devez **installer 🤗 Transformers à partir du code source** dans un nouvel environnement virtuel :
```

File: docs/source/fr/run_scripts_fr.md [hunk 3]
```diff
@@ -0,0 +1,355 @@
...
+	</ul>
+</details>
+
+Ensuite, changez votre clone actuel de  🤗 Transformers pour une version spécifique, comme par exemple v3.5.1 :
+
+```bash
+git checkout tags/v3.5.1
+```
+
+Après avoir configuré la bonne version de la bibliothèque, accédez au dossier d'exemple de votre choix et installez les prérequis spécifiques à l'exemple.
```
Comment (ArthurZucker) [inline docs/source/fr/run_scripts_fr.md:81]:
```suggestion
Après avoir configuré la bonne version de la librairie, accédez au dossier d'exemple de votre choix et installez les prérequis spécifiques à l'exemple.
```

File: docs/source/fr/run_scripts_fr.md [hunk 4]
```diff
@@ -0,0 +1,355 @@
...
+```bash
+pip install -r requirements.txt
+```
+
+## Exécuter un script
+
+<frameworkcontent>
+<pt>
+
+Le script exemple télécharge et prétraite un jeu de données à partir de la bibliothèque 🤗 [Datasets](https://huggingface.co/docs/datasets/). Ensuite, le script affine un ensemble de données à l'aide de [Trainer](https://huggingface.co/docs/transformers/main_classes/trainer) sur une architecture qui prend en charge la tâche de résumé. L'exemple suivant montre comment ajuster le modèle [T5-small](https://huggingface.co/google-t5/t5-small) sur les données [CNN/DailyMail](https://huggingface.co/datasets/cnn_dailymail). Le modèle T5 nécessite un argument supplémentaire source_prefix en raison de la façon dont il a été entraîné. Cette invite permet à T5 de savoir qu'il s'agit d'une tâche de résumé.
```
Comment (ArthurZucker) [inline docs/source/fr/run_scripts_fr.md:92]:
```suggestion
Le script d'exemple télécharge et prétraite un jeu de données à partir de la bibliothèque 🤗 [Datasets](https://huggingface.co/docs/datasets/). Ensuite, le script affine un ensemble de données à l'aide de [Trainer](https://huggingface.co/docs/transformers/main_classes/trainer) sur une architecture qui prend en charge la tâche de résumé. L'exemple suivant montre comment ajuster le modèle [T5-small](https://huggingface.co/google-t5/t5-small) sur les données [CNN/DailyMail](https://huggingface.co/datasets/cnn_dailymail). Le modèle T5 nécessite un argument supplémentaire `source_prefix` en raison de la façon dont il a été entraîné. Cette invite permet à T5 de savoir qu'il s'agit d'une tâche de résumé.
```

File: docs/source/fr/run_scripts_fr.md [hunk 5]
```diff
@@ -0,0 +1,355 @@
...
+```
+</tf>
+</frameworkcontent>
+
+## Entraînement distribué et précision mixte
+
+[Trainer](https://huggingface.co/docs/transformers/main_classes/trainer) prend en charge l'entraînement distribué et la précision mixte, ce qui signifie que vous pouvez également les utiliser dans un script. Pour activer ces deux fonctionnalités :
+
+- Ajoutez l'argument fp16 pour activer la précision mixte.
+- Définissez le nombre de GPU à utiliser avec l'argument nproc_per_node.
```
Comment (ArthurZucker) [inline docs/source/fr/run_scripts_fr.md:133]:
```suggestion
- Définissez le nombre de GPU à utiliser avec l'argument `nproc_per_node`.
```

File: docs/source/fr/run_scripts_fr.md [hunk 6]
```diff
@@ -0,0 +1,355 @@
...
+```
+
+Les scripts TensorFlow utilisent une Strategie en Miroir [`MirroredStrategy`](https://www.tensorflow.org/guide/distributed_training#mirroredstrategy) pour l'entraînement distribué, et vous n'avez pas besoin d'ajouter d'arguments supplémentaires au script d'entraînement. Le script TensorFlow utilisera plusieurs GPU par défaut s'ils sont disponibles.
+
+## Exécuter un script sur un TPU 
+
+<frameworkcontent>
+<pt>
+
+Les unités de traitement de tenseurs (UTT) (TPU) sont spécialement conçues pour accélérer les performances. PyTorch prend en charge les TPU avec le compilateur de deep learning [XLA](https://www.tensorflow.org/xla). Pour utiliser un TPU, lancez le script xla_spawn.py et utilisez l'argument num_cores pour définir le nombre de cœurs TPU que vous souhaitez utilise
```
Comment (ArthurZucker) [inline docs/source/fr/run_scripts_fr.md:159]: unités de traitement de tenseur sounds super strange

File: docs/source/fr/run_scripts_fr.md [hunk 7]
```diff
@@ -0,0 +1,355 @@
...
+    --num_train_epochs 3 \
+    --do_train \
+    --do_eval
+```
+</tf>
+</frameworkcontent>
+
+## Exécuter un script avec 🤗 Accelerate 
+
+🤗 [Accelerate](https://huggingface.co/docs/accelerate) est une bibliothèque uniquement pour PyTorch qui offre une méthode unifiée pour entraîner un modèle sur plusieurs types de configurations (CPU uniquement, plusieurs GPU, TPU) tout en maintenant une visibilité complète sur la boucle d'entraînement PyTorch. Assurez-vous que vous avez installé 🤗 Accelerate si ce n'est pas déjà le cas.
```
Comment (ArthurZucker) [inline docs/source/fr/run_scripts_fr.md:198]: entrainer et inférer

File: docs/source/fr/run_scripts_fr.md [hunk 8]
```diff
@@ -0,0 +1,355 @@
...
+
+## Utiliser un jeu de données personnalisé 
+
+Le script de résumé prend en charge les jeux de données personnalisés tant qu'ils sont au format CSV ou JSON Line. Lorsque vous utilisez votre propre jeu de données, vous devez spécifier plusieurs arguments supplémentaires :
+
+- `train_file` et `validation_file` spécifient le chemin vers vos fichiers d'entraînement et de validation.
+- `text_column` est le texte d'entrée à résumer.
+- `summary_column` est le texte cible à produire.
+
+Un exemple de script de résumé utilisant un ensemble de données personnalisé ressemblerait à ceci :
```
Comment (ArthurZucker) [inline docs/source/fr/run_scripts_fr.md:236]:
```suggestion
Un exemple de script de résumé utilisant un ensemble de données personnalisé ressemblerait à ceci :
```
un script de résumé n'ap as de sense à revoir

**Review by ArthurZucker — COMMENTED** [review]
thanks for translating

File: docs/source/fr/run_scripts_fr.md (hunk 6 above)
Comment (jadechoghari) [reply docs/source/fr/run_scripts_fr.md:159]: true, mais je pense que c'est correct:
https://phoenixnap.fr/glossaire/qu%27est-ce-qu%27une-unit%C3%A9-de-traitement-de-tenseur

File: docs/source/fr/run_scripts_fr.md (hunk 7 above)
Comment (jadechoghari) [reply docs/source/fr/run_scripts_fr.md:198]: D'accord, vous pouvez la modifier
merci!

**Review by ArthurZucker — APPROVED** [review]

**Pull request merged**